pkg/engine/utils_test.go

```diff
@@ -880,7 +880,7 @@ func TestMatchesResourceDescription(t *testing.T) {
 		resource, _ := utils.ConvertToUnstructured(tc.Resource)
 
 		for _, rule := range policy.Spec.Rules {
-			err := MatchesResourceDescription(*resource, rule, tc.AdmissionInfo, []string{}, nil)
+			err := MatchesResourceDescription(*resource, rule, tc.AdmissionInfo, []string{}, nil, "")
```

Comment:
The extra parameter in the function, you added policy type right! why are we not sending here ?

Reply:
Also where are the test cases added for the newly added parameter ?